src/components/SummaryCard.tsx

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import icon from '../assets/icon-music.svg';
+import PricePlan from './PricePlan';
+
 import hero from '../assets/illustration-hero.svg';
 
+type PricePlanType = {
+  type: 'annual' | 'monthly';
+  amount: number;
+};
+
+type PricePlanMap = {
+  annual: PricePlanType;
+  monthly: PricePlanType;
+};
+
+const pricePlan: PricePlanMap = {
+  annual: {
+    type: 'annual',
+    amount: 59.99,
+  },
+  monthly: {
+    type: 'monthly',
+    amount: 6.99,
+  },
+};
+
 const SummaryCard = () => {
+  const [selectedPlan, setSelectedPlan] = React.useState(pricePlan.annual);
+
   return (
     <Card>
```

Comment:
I think a PriceCard would be a suitable refactor. Let's say if we are going to create a Pricing page.

Reply:
We can loop through the available plans and show each one of them